spec/models/peer_review_spec.rb

```diff
@@ -27,7 +27,10 @@
       expect(PeerReview.all.size).to eq 0
 
       assignment_with_pr = create(:assignment_with_peer_review_and_groupings_results)
-      PeerReviewsController.new.perform_random_assignment(assignment_with_pr.pr_assignment, 2)
+      selected_reviewer_group_ids = assignment_with_pr.pr_assignment.groupings.map { |g| g.id }
+      selected_reviewee_group_ids = assignment_with_pr.groupings.map { |g| g.id }
+      PeerReviewsController.new.perform_random_assignment(assignment_with_pr.pr_assignment, 2,
```

Comment:
Sorry, I didn't catch this before. This needs to be a controller spec test, not a model test.